easydigitaldownloads/easy-digital-downloads
# Define default/fresh admin color scheme

Fixes #8229

To test:
1. Visit a screen such as Exports.
2. Make sure that any select inputs using Chosen highlight a selection properly using the default (fresh) admin color scheme.
3. Make sure that the color is still correct when using an alternative color scheme.
